.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "needs-triage"]
+projects: ["saltstack/51"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    validations:
+      required: true
+    attributes:
+      label: What happened?
+      description: A clear and concise description of what the bug is. Please provide relevant configs and/or SLS files (be sure to remove sensitive info. There is no general set-up of Salt.) Please be as specific as possible and give set-up details, and a concise description of what you expected to happen.
+      value: "A bug happened!"
+  - type: dropdown
+    id: salt-type
+    validations:
+      required: true
+    attributes:
+      label: Type of salt install
+      multiple: false
+      description: What type of Salt installation is this?
+      options:
+        - Official deb
+        - Official rpm
+        - Official exe
+        - Official msi
```

Comment:
Maybe "Official msi/exe"

Reply:
I think having them separate is a good thing.